Directory.Build.props

```diff
@@ -18,7 +18,7 @@
     <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
     <RepositoryUrl>https://github.com/fsprojects/FSharp.Data.GraphQL</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
-    <Version>2.0.0</Version>
+    <Version>2.1.0</Version>
```

Comment:
That would be a major change because projects targeting .net6 and referencing FSharp.Data.GraphQL version `2.0.0` will be in for a surprise when they update it to `2.1.0` thinking not much about it.
We know this from Microsoft and it's just an annoying behavior.

Reply:
New push has 3.0.0 in that place